R/response_process.R

```diff
@@ -107,40 +113,38 @@ check_for_json <- function(resp, call = caller_env()) {
     return(invisible(resp))
   }
 
-  content <- httr::content(resp, as = "text")
   gargle_abort_request_failed(
-    c(
-      gargle_map_cli(
-        type,
-        template = "Expected content type {.field application/json}, not \\
-                    {.field <<x>>}."
-      ),
-      "*" = obfuscate(content, first = 197, last = 0)
```

Comment:
Note: this is where the `Could not parse cli `{}` expression: ...` error was coming from (#254). When the `content` is html, it can contain curly braces (and, indeed, in this case, did), which then confuses cli when it processes the error message bullets.

There is a solution, which is to refer to the `{`-and-`}`-containing string from a template:

https://cli.r-lib.org/reference/inline-markup.html#escaping-and-

However, I decided not to do that, but instead to make it much easier to see the full html. Previously, my only direct experience of html errors was from the token info endpoints, which seemed rather obscure and the examples I knew about produced a relatively small amount of html.

But the error seen in #254 is actually important for the user (429 too many requests, from googledrive) and involved considerably more html (so a wee little excerpt didn't reveal anything useful).

Reply:
This pitfall is also exactly described here:

https://rlang.r-lib.org/reference/topic-condition-formatting.html#transitioning-from-abort-to-cli-abort-